test/evergreen.yml

```diff
@@ -554,6 +554,31 @@ functions:
           done
         done
 
+  "generic-perf-test":
+    # Parameterised using the 'perf-test-name' and 'maxruns' variables
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/bench/wtperf/wtperf_run_py"
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+          pip3 install psutil
+          python3 wtperf_run.py -p ../../../cmake_build/bench/wtperf/wtperf -t ../runners/${perf-test-name}.wtperf -v -ho WT_TEST -m ${maxruns} -o out.json
```

Comment:
`${python_binary|python3} ` instead of just `python3`?

Reply:
I agree, `python3` should be changed. I think it should be to `${python_binary}` as I think it's better not to provide a default value.